### ISSUE TITLE
httpyac cli does not exit when invoked with the same file twice

### ISSUE THREAD

**Issue by jhonnycano** [AnWeber/httpyac#384]
Steps to reproduce:

* Create `test.http` file with following content:
```
POST https://httpbin.org/anything
```
* Invoke httpyac from command line:
```
httpyac -a test.http test.http
```

The execution won't finish.

Note: Not that helps much, but this worked in a previous version of at least 7 months ago. I have several scenarios like the following:
```
httpyac -a file1.http file2.http file1.http
```
in which file1 queries my backend, then I execute file2 (to insert and change the assertion variable) and execute again the file1 to check.

Thanks a lot

**Comment by AnWeber** [comment]
seems to be an issue in [lib hookpoint in setUniqueId](https://github.com/AnWeber/hookpoint/blob/3a94dcd2c7812cb59510f4d33bc9ba66036f0d5e/src/hooks/sortSet.ts#L36). This method is stuck in an infinite loop.

**Comment by AnWeber** [comment]
Issue is fixed with v5.10.2. I would never have thought of using the same file twice.
